gradle/spotless.gradle

```diff
@@ -0,0 +1,7 @@
+spotless {
+  java {
+    removeUnusedImports()
```

Comment:
why do not use any format such as eclipseFormatter or googleJavaFormat?

https://github.com/diffplug/spotless/blob/master/plugin-gradle/README.md#applying-to-java-source-with-google-java-format

Reply:
It's also possible solution, I think. I just wanted to keep the diff simple.

@spotbugs/everyone if you don't care, I will introduce the latest googleJavaFormat.